indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -233,6 +245,12 @@ public TaskStatus call()
                               command.add(String.format("-Ddruid.host=%s", childHost));
                               command.add(String.format("-Ddruid.port=%d", childPort));
 
+                              if(config.isSeparateIngestionEndPoint()) {
+                                command.add(String.format("-Ddruid.indexer.task.chathandler.service=%s", "druid/chathandler"));
```

Comment:
I checked putting invalid path as serviceName will not help, `makeCanonicalServiceName` method in `CuratorServiceUtils` class replaces all "\" with ":"
However I have put another character `\u0001` (invalid for zk) in the String and it will fail if this name is used. Also I have added some UTs for `ServiceAnnouncer` to test the behavior. Please review.